src/components/Login.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import { instanceOf } from 'prop-types';
+import { Cookies, withCookies } from '../utils/cookie_utils';
+import { logInWithEmailAndPassword, useNavigate } from '../utils/auth_utils';
+// import { logInWithEmailAndPassword , signInWithGoogle, useNavigate } from '../utils/auth_utils';
+
+const Login = ({ cookies }) => {
```

Comment:
⚠️ **[eslint]** <react-refresh/only-export-components> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Fast refresh only works when a file only exports components. Move your component(s) to a separate file.